sale_documents_comments/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================================
+Comments for sale documents (order, picking and invoice)
```

Comment:
- technical module name (folder name) according guidelines must be in singular form: i.e. sale_document_comment.
  Technical name is repeated in other modules for many purpose like inherit views: singular is easier to memorize. Else is it ? sale_document_comments, sale_documents_comments, sale_documents_comment 

- Why friendly module name is so far from technical name (Comments for sale documents => sale_documents_comments)

- maybe consider name as: `sale_workflow_comment` to express in 1 word: sale, picking, invoice